Return error conditions including in the case of not changing a file.
--- a/vbrfixc/vbrfix.cpp
+++ b/vbrfixc/vbrfix.cpp
@@ -73,6 +73,9 @@
     endFix();
     return false;
   }
+  catch (...) {
+    return false;
+  }
   return true;
 }
 void VbrFix::endFix(){
@@ -113,7 +116,10 @@
   catch(const char* err){
     if(err!=NULL)loge(err);
     endFix();
-    return false;
+    exit(-1);//return false;
+  }
+  catch (...) {
+    exit(-1);//return false;
   }
   return false;
 }
